walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css

```diff
@@ -1,25 +1,67 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+body {
+  font-family: Arial, sans-serif;
+  padding: 4px;
+  text-align: center;
 }
 
-.image-thumbnails {
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: center;
+h1 {
+  text-align: center;
+  font-weight: normal;
```

Comment:
Please [sort css attributes alphabetically](https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order) here and below.